### ISSUE TITLE
Opening properties of the `GWENT: The Witcher Card Game` crashes Steam

### ISSUE THREAD

**Issue by jqadev** [ValveSoftware/steam-for-linux#7238]
#### [Your system information](https://gist.github.com/jqadev/8a892c8f8b5145169b4d5ae8c142c7c9)

* Steam client version (build number or date): `Jun 4 2020, at 05:51:01`, steam-runtime_0.20200505.0
* Distribution (e.g. Ubuntu): `Ubuntu 20.04 LTS`
* Opted into Steam client beta?: `Beta Jun 30 2020, at 04:15:14` is also affected
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Steam crashes/closes when I try to open properties of the `GWENT: The Witcher Card Game`.
Other games in my library aren't affected by this issue. 
When I created the report https://github.com/ValveSoftware/Proton/issues/3890 on May 19 2020 I used steam-runtime_0.20200417.0 and then this issue didn't exist so it is a regression.

#### Steps for reproducing this issue:

1. Open Steam
2. Go to Library
3. Press right mouse button on `GWENT: The Witcher Card Game` 
4. Click Properties

**Comment by jqadev** [comment]
The reason of this issue was `Proton-5.8-GE-2-MF` set by me as a Steam Play compatibility tool on May 19 2020.
I uninstalled the game few days later but the option stayed set.
A few days later I deleted directory `Proton-5.8-GE-2-MF` from  `~/.steam/steam/compatibilitytools.d/`.
Today when I had a problem with opening properties I made `~/.steam/steam/compatibilitytools.d/` directory empty by removing `Proton-5.9-GE-2-MF` I had there.
Then I restarted Steam and opening properties of the game started to work.

This case should be handled by Steam to prevent crashes.
